app/src/main/java/com/example/benchproject/domain/popular/movies/entity/MovieModel.kt

```diff
@@ -1,9 +1,20 @@
 package com.example.benchproject.domain.popular.movies.entity
 
+import com.example.benchproject.app.popular.movies.entity.Movie
+
 data class MovieModel(
     val id: Int,
     val name: String,
     val rating: Double,
     val releaseDate: String,
     val imageUrl: String
 )
+fun List<MovieModel>.mapToUI() = this.map { movieModel ->
```

Comment:
You are using `toDomain` for ui->domain mapping so you can use `toUi` for domain->ui mapping instead of `mapToUI` :D it will be more consistent

Reply:
Of course you could do it in another PR

Reply:
I created a project and I added a new [task](https://github.com/Nataniel-Antosik/Bench_Project/issues/36) in `To Do` column to change all `mapToUI` on `toUi` I will change all of them when this PR and #29 will be merged to `develop`

Reply:
I change this `mapToUI` in previous PR and in this I do the same therefore I don't must create new PR for this changes